validate/boolean/src/index.ts

```
@@ -0,0 +1,141 @@
+        field,
+        config.customErrorMessages?.nullValue ||
+          'Value cannot be null or undefined'
+      )
+      break
+    case 'false':
+      record.set(field, false)
+      break
+    case 'true':
+      record.set(field, true)
+      break
+    case 'skip':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)